vita3k/modules/SceNpTrophy/SceNpTrophy.cpp

```diff
@@ -143,7 +148,7 @@ EXPORT(int, sceNpTrophyGetGameInfo, np::trophy::ContextHandle context_handle, Sc
             }
         }
 
-        // TODO: set title and description
+        details->size = sizeof(SceNpTrophyGameDetails);
```

Comment:
This line should be removed as `details->size` is already set and checked.

Reply:
i have done all change requuest